demo/kubernetes/rook-flexvolume.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
```

Comment:
yes this is the intention. This was just a quick way to deploy and test it. I will change it so that the operator does this